simulation/revolve2/simulation/simulator/_record_settings.py

```diff
@@ -13,3 +13,5 @@ class RecordSettings:
 
     width: Optional[int] = None
     height: Optional[int] = None
+
+    # offscreen: bool = False << For future implementation (student-proof?)
```

Comment:
Can be put in , but default set to false, also a docstrinng explaining it would be nice

Reply:
Ok now thinking about it its probably better to open an issue for this and remove the comment here